src/components/form-sections/HasNextEnrollment/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
src/components/form-sections/HasNextEnrollment/index.js => src/components/form-sections/HasNextEnrollment.js ?

Reply:
My consistenciphilia pinch me a bit, because this one is not an actual section but rather a notification. I'd rather put this alongside with src/components/Form/EnrollmentHasCopiesNotification.js, don't you think ?